gensim/models/wrappers/fasttext.py

```diff
@@ -137,6 +137,12 @@ def __contains__(self, word):
             char_ngrams = compute_ngrams(word, self.min_n, self.max_n)
             return any(ng in self.ngrams for ng in char_ngrams)
 
+    @classmethod
+    def load_word2vec_format(cls, fname, fvocab=None, binary=False, encoding='utf8', unicode_errors='strict',
```

Comment:
One tiny change I might suggest here would be to use *args and **kwargs here. I like following this practice in situations where the subclass make use of the function params itself. It is useful because if any params in the parent class change in the future, the subclass function signature doesn't have to be modified.